raiden/tests/integration/fixtures/raiden_network.py

```diff
@@ -16,6 +16,15 @@
 from raiden.tests.utils.tests import shutdown_apps_and_cleanup_tasks
 
 
+def wait_for_min_blocks(raiden, web3):
```

Comment:
I don't think this function is needed. We already got something similar in the `waiting.py` module [here](https://github.com/raiden-network/raiden/blob/a6e95b986892fb8185a14ca91a093eb847b71b0d/raiden/waiting.py#L19)

Reply:
`waiting.wait_for_blocks` cannot be used here. We need to wait for the chain to grow long enough before starting the apps, so we don't have a `RaidenService` instance to pass to it yet.

Reply:
Hmm if we don't have a Raiden instance then what is the `raiden` argument for? Which reminds me ... put the types of the arguments ^_^


Also in any case this should probably be in the waiting.py ?

Reply:
Good point, I was confused here. We have a `RaidenService` object at this point, but its `wal` attribute is not yet initialized. That's why `wait_for_block` will not work.